ports/stm32/boards/Passport/modules/flows/login_flow.py

```diff
@@ -18,18 +18,18 @@
 class LoginFlow(Flow):
     def __init__(self):
         self.pin = ''
-        super().__init__(initial_state=self.enter_pin, name='LoginFlow')
-
-    async def enter_pin(self):
 
         device_name = common.settings.get('device_name', None)
         if device_name is not None:
             device_name = device_name.upper()
+        self.statusbar = {'title': (device_name or 'PASSPORT')}
```

Comment:
Any reason for this to be saved as a member instead of passed directly to the base class constructor?